gflownet/envs/crystals/spacegroup.py

```diff
@@ -539,8 +559,61 @@ def get_ref_index(self, state: List[int] = None) -> int:
             state = self.state
         return sum([int(s > 0) * f for s, f in zip(state, self.ref_state_factors)])
 
-    def get_all_terminating_states(self) -> List[List]:
+    def set_n_atoms(self, composition):
```

Comment:
The only reason to return was to make it explicit in the `__init___` what it is being set, with `self.n_atoms, self.compatibility_stoichiometry_dict = self.set_n_atoms(n_atoms)`. But I agree with your suggestion. I will change it.

Reply:
For making it explicit, that I think is fine as well, in that case I would just rename it from `set_*` to `get_*`.

Reply:
Updated in [e4efd50](https://github.com/alexhernandezgarcia/gflownet/pull/148/commits/e4efd5098d00989acf12d210d6a57bd367791ee3).

Reply:
I have sacrificed explicitness in the end, since 
```
# Set dictionary of compatibility with number of atoms
self.set_n_atoms_compatibility_dict(n_atoms)
```
is explicit enough, I think.